### PR TITLE
Changes Marish

### PR DESCRIPTION
**Pull request by EtheoBoxxman**
## About The Pull Request
Removes the syllables from marish and makes the language behave like echosong. Also gives the language a less intrusive font color!
No more drowning out entire chat logs with casual conversation in marish.
Its also slightly more lore accurate, involving the ears as part of its verbage as well. 
## Changelog
:cl: Etheo
fix: Fixed Marish to be better and more lore accurate.
/:cl:

### CODE REVIEW

**Comment by EtheoBoxxman** [comment]
Github and VSC misbehaving as usual and copying ancient PR commits to new ones. :/
Fortunately none of it is actually in the code changes, lol.

**Comment by Kashargul** [comment]
css files for the chat are to be kept in sync with upstream, furthermore the language also needs to be added to the byond internal chat css.

**Comment by EtheoBoxxman** [comment]
> css files for the chat are to be kept in sync with upstream, furthermore the language also needs to be added to the byond internal chat css.

This explains nothing to me.

File: tgui/packages/tgui-panel/styles/tgchat/chat-dark.scss [hunk 1]
```diff
@@ -1429,6 +1429,10 @@ $border-width-px: $border-width * 1px;
   line-height: 0.3;
 }
 
+.marish {
```
Comment (Kashargul) [inline tgui/packages/tgui-panel/styles/tgchat/chat-dark.scss:1432]: This is an synced upstream file. The change needs to be made on Virgo.

File: tgui/packages/tgui-panel/styles/tgchat/chat-light.scss [hunk 2]
```diff
@@ -1462,6 +1462,10 @@ $border-width-px: $border-width * 1px;
   line-height: 0.3;
 }
 
+.marish {
+  color: #652868;
+}
+
```
Comment (Kashargul) [inline tgui/packages/tgui-panel/styles/tgchat/chat-light.scss:1468]: This is an synced upstream file. The change needs to be made on Virgo.

File: tgui/packages/tgui-panel/styles/tgchat/chat-vchatdark.scss [hunk 3]
```diff
@@ -1429,6 +1429,10 @@ $border-width-px: $border-width * 1px;
   line-height: 0.3;
 }
 
+.marish {
+  color: #652868;
+}
+
```
Comment (Kashargul) [inline tgui/packages/tgui-panel/styles/tgchat/chat-vchatdark.scss:1435]: This is an synced upstream file. The change needs to be made on Virgo.

File: tgui/packages/tgui-panel/styles/tgchat/chat-vchatlight.scss [hunk 4]
```diff
@@ -1460,6 +1460,10 @@ $border-width-px: $border-width * 1px;
   line-height: 0.3;
 }
 
+.marish {
+  color: #652868;
+}
+
```
Comment (Kashargul) [inline tgui/packages/tgui-panel/styles/tgchat/chat-vchatlight.scss:1466]: This is an synced upstream file. The change needs to be made on Virgo.

**Review by Kashargul — CHANGES_REQUESTED** [review]
interface\stylesheet.dm

 is missing the changes, here as well, changes done directly on virgo.

File: code/modules/mob/language/station_vr.dm [hunk 5]
```diff
@@ -179,6 +186,10 @@
 /datum/language/echosong/scramble(var/input, var/list/known_languages)
 	return stars(input)
```
Comment (Kashargul) [inline code/modules/mob/language/station_vr.dm:188]: missing comments, why was this upstream file edits just to move some code around?

File: code/modules/mob/language/station_vr.dm [hunk 6]
```diff
@@ -105,14 +105,21 @@
```
Comment (Kashargul) [inline code/modules/mob/language/station_vr.dm:105]: //CHOMPAdd Start


all the way down to line 138 then there //CHOMPAdd End

**Review by Kashargul — CHANGES_REQUESTED** [review]

File: code/modules/mob/language/station_vr.dm (hunk 6 above)
Comment (EtheoBoxxman) [reply code/modules/mob/language/station_vr.dm:105]: Already have a comment for it.

File: code/modules/mob/language/station_vr.dm (hunk 5 above)
Comment (EtheoBoxxman) [reply code/modules/mob/language/station_vr.dm:188]: I moved it because it was unorganized.

**Review by Kashargul — APPROVED** [review]

**Comment by Kazzc** [comment]
Alright, awesome. This is much better!

**Pull request merged**